kinley-matt/styles/modules.css

```diff
@@ -0,0 +1,78 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+.icon-menu:before {
```

Comment:
You do not need to style the `:before` pseudo-class in order to change the hamburger menu's styling, just adding rules for `.icon-menu` will work.